支持go 1.18 编译

go 1.18 编译问题

```
➜  code git:(master) ✗ go version
go version go1.18 darwin/arm64
➜  code git:(master) ✗ go build
# github.com/go-zoox/feishu/message/content
../../../../go/pkg/mod/github.com/go-zoox/feishu@v1.3.9/message/content/content.go:4:16: interface contains type constraints
```

--- a/go.mod
+++ b/go.mod
@@ -9,7 +9,7 @@ require (
 	github.com/go-zoox/cli v1.2.0
 	github.com/go-zoox/core-utils v1.2.7
 	github.com/go-zoox/debug v1.0.1
-	github.com/go-zoox/feishu v1.3.9
+	github.com/go-zoox/feishu v1.3.10
 	github.com/go-zoox/fs v1.3.9
 	github.com/go-zoox/logger v1.4.4
 	github.com/go-zoox/retry v1.0.3

